pkg/controller/controller_utils.go

```diff
@@ -980,7 +980,7 @@ func FilterActivePods(logger klog.Logger, pods []*v1.Pod) []*v1.Pod {
 		if IsPodActive(p) {
 			result = append(result, p)
 		} else {
-			logger.V(4).Info("Ignoring inactive pod", "pod", klog.KObj(p), "phase", p.Status.Phase, "deletionTime", klog.SafePtr(p.DeletionTimestamp))
+			logger.V(5).Info("Ignoring inactive pod", "pod", klog.KObj(p), "phase", p.Status.Phase, "deletionTime", klog.SafePtr(p.DeletionTimestamp))
```

Comment:
Why do we modify the log level in code instead of modifying the component's log output level? 🤔

Reply:
@googs1025 Since most of the logs in the kube-controller-manager are at V(4) level, it's common to enable V(4) logs for troubleshooting. However, in the batch job scenario, if a large number of Jobs are in the Completed state and their ttlSecondsAfterFinished is set too high, it can generate 1.8GB of logs per minute, which may fill up the disk on the machine.